### PR TITLE
Add a Dockerfile to run gitsome in a Docker container

### PR DESCRIPTION
**Pull request by l0rd**
Signed-off-by: Mario Loriedo <mario.loriedo@gmail.com>

image

### CODE REVIEW

**Comment by donnemartin** [comment]
Hi @l0rd, thanks for the pull request!

Could you also update the README with Docker install instructions?

* [Index](https://github.com/donnemartin/gitsome#installation-and-tests)
    * New index entry **Docker Installation** after **Virtual Environment Installation**
* [Docker Installation](https://github.com/donnemartin/gitsome#virtual-environment-installation)
    * New section after the **Virtual Environment Installation**
* Your change to "Available Platforms" could also hyperlink **Docker** to your new **Docker Installation** section

**Comment by l0rd** [comment]
@donnemartin I've done what you've asked. 

Note that I'm using [my Docker hub repository](https://hub.docker.com/r/mariolet/gitsome/builds/) in the doc: 

   $ docker run -ti --rm mariolet/gitsome

You will probably want to use another Docker hub repository with [an automated build](https://docs.docker.com/docker-hub/builds/).

File: README.md [hunk 1]
```diff
@@ -121,7 +121,8 @@ For no color, set the value(s) to `None`.  `white` can appear as light gray on s
 
 ## Available Platforms
 
-`gitsome` is available for Mac, Linux, Unix, and [Windows](#windows-support).
+`gitsome` is available for Mac, Linux, Unix, [Windows](#windows-support) and
+[Docker](#running-as-docker-container).
```
Comment (donnemartin) [inline README.md]: Can we combine lines 124 and 125 into a single line?

File: README.md [hunk 2]
```diff
@@ -176,6 +177,7 @@ For no color, set the value(s) to `None`.  `white` can appear as light gray on s
 * [Installation](#installation)
     * [Pip Installation](#pip-installation)
     * [Virtual Environment Installation](#virtual-environment-installation)
+    * [Running as Docker container](#running-as-docker-container)
```
Comment (donnemartin) [inline README.md]: Can we capitalize "Container"?

File: README.md [hunk 3]
```diff
@@ -548,6 +550,34 @@ To deactivate the `gitsome` `virtualenv`, run:
 
     $ deactivate
 
+### Running as Docker Container
```
Comment (donnemartin) [inline README.md]: Should this be "Running as a Docker Container"?  If we make this change, please update line 180

**Comment by donnemartin** [comment]
@l0rd Thank you!

**Pull request merged**